docs/pages/index.tsx

```diff
@@ -1,47 +1,73 @@
 import type { NextPage } from 'next';
 import Head from 'next/head';
-import Link from 'next/link'
-import styles from '../styles/Home.module.css';
+import Link from 'next/link';
 
 const Home: NextPage = () => {
   return (
-    <div className={styles.container}>
+    <div className="min-w-0 max-w-2xl flex-auto px-4 py-16 lg:max-w-none lg:pr-0 lg:pl-8 xl:px-16">
       <Head>
         <title>CooperTS</title>
         <meta name="description" content="Documentation for CooperTS by ExecOnline" />
       </Head>
 
-      <main className={styles.main}>
-        <h1 className={styles.title}>
+      <main>
+        <p className="lead">
           Welcome to <a href="#">CooperTS!</a>
-        </h1>
+        </p>
```

Comment:
Can we change this back to an h1? We need a single h1, and we have 3 h2s, so our hierarchal order is askew.